libs/components/src/lib/header/header.template.ts

```diff
@@ -23,19 +21,26 @@ const getClasses = ({ fixed, alternate, elevated }: Header) => classNames(
 export const headerTemplate: (
 	context: ElementDefinitionContext,
 	definition: FoundationElementDefinition
-) => ViewTemplate<Header> = () => {
+) => ViewTemplate<Header> = (context) => {
+	const elevationTag = context.tagFor(Elevation);
+
 	return html<Header>`
-    <header class="${getClasses}" part="${(x) => x.alternate ? 'vvd-theme-alternate' : ''}">
-        <section class="header-content start" id="navigation">
-          <slot></slot>
-          ${when(x => x.heading, html<Header>`<span class="heading">${x => x.heading}</span>`)}
-        </section>
-        <section class="header-content end" id="actions" role="toolbar">
-          <slot name="actionItems"></slot>
-        </section>
-    </header>
+    <${elevationTag} dp="4" ?no-shadow=${x => !x.elevationShadow}>
+      <header class="base" part="base">
+        <!-- a container is needed to distinguish the surface background color of the
+        element from its shadow when applying elevation with alternate -->
+        <div class="container" part="${getPartAlternate}">
+          <section class="header-content start">
```

Comment:
```suggestion
          <section class="header-content">
```
you removed the classes in css - so no use for it here :)